spiped/dispatch.c

```diff
@@ -247,3 +253,23 @@ dispatch_shutdown(void * dispatch_cookie)
 		dnsthread_kill(A->T);
 	free(A);
 }
+
+/**
+ * request_shutdown(dispatch_cookie):
+ * Requests a shutdown: Stop accepting new connections and notify once
+ * every existing connection ended.
+ */
+void
+request_shutdown(void * dispatch_cookie)
```

Comment:
You are talking about the function body, right?
Am 14.05.2016 23:31 schrieb "Colin Percival" <notifications@github.com>:

> In spiped/dispatch.c
> <https://github.com/Tarsnap/spiped/pull/51#discussion_r63282328>:
>
> > @@ -247,3 +253,23 @@ dispatch_shutdown(void * dispatch_cookie)
> >  		dnsthread_kill(A->T);
> >  	free(A);
> >  }
> > +
> > +/**
> > + * request_shutdown(dispatch_cookie):
> > + * Requests a shutdown: Stop accepting new connections and notify once
> > + * every existing connection ended.
> > + */
> > +void
> > +request_shutdown(void * dispatch_cookie)
>
> Needs comments explaining what we're doing.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Tarsnap/spiped/pull/51/files/b760ac699dd7f655055afe5840f27b44c578da81#r63282328>
>